### PR TITLE
updated docs to include the new copilot related meeting policy settings.

### PR DESCRIPTION
**Pull request by boboPD**

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -1115,6 +1115,44 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Copilot
+This setting allows the admin to choose if Copilot will be enabled with a persisted transcript or a non-persisted transcript.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:1119]:
```suggestion
This setting allows the admin to choose whether Copilot will be enabled with a persisted transcript or a non-persisted transcript.
```

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 2]
```diff
@@ -1115,6 +1115,44 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: EnabledWithTranscript
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AutomaticallyStartCopilot
+This setting gives admins the ability to auto-start copilot.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:1138]:
```suggestion
This setting gives admins the ability to auto-start Copilot.
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 3]
```diff
@@ -1188,6 +1188,44 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Copilot
+This setting allows the admin to choose if Copilot will be enabled with a persisted transcript or a non-persisted transcript.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:1192]:
```suggestion
This setting allows the admin to choose whether Copilot will be enabled with a persisted transcript or a non-persisted transcript.
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 4]
```diff
@@ -1188,6 +1188,44 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: EnabledWithTranscript
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AutomaticallyStartCopilot
+This setting gives admins the ability to auto-start copilot.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:1211]:
```suggestion
This setting gives admins the ability to auto-start Copilot.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @boboPD, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by boboPD** [comment]
@yogkumgit these changes are ready to be merged in now. Is there anything else that you need from me?

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**